### DISCUSSION TITLE
Blank start/end point on Drives dashboard

### DISCUSSION THREAD

**Discussion by ArvinLiYongJun** [teslamate-org/teslamate#4111] [Q&A]
Since I started using Teslamate, the start and destination in Drives have always been blank. I want to know why and how to solve it.
![Snipaste_2024-08-01_10-22-00](https://github.com/user-attachments/assets/f88e94ad-2680-40d6-b28e-902210879f62)

**Comment by cwanja** [comment]
Going to guess main land China? If so, Openstreet, which is used to provide geo locations, is blocked in China.

**Reply by cwanja** [reply]
See #3912 and #3960
